code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -75,3 +75,32 @@
 	embedding = list(embed_chance=80, fall_chance=1, jostle_chance=3, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=3, jostle_pain_mult=2, rip_time=14)
 	embed_falloff_tile = -3
 	shrapnel_type = /obj/item/stack/rods
+
+// AKM Boollets
+
+/obj/projectile/bullet/mm712x82
+	name = "7.12x82mm bullet"
+	damage = 40
+	armour_penetration = 5
+	wound_bonus = -40
+	wound_falloff_tile = 0
+
+/obj/projectile/bullet/mm712x82/ap
+	name = "7.12x82mm armor-piercing bullet"
+	damage = 35
+	wound_bonus = -45
+	armour_penetration = 45
+	bare_wound_bonus = -10
+
+/obj/projectile/bullet/mm712x82/hp
+	name = "7.12x82mm hollow-point bullet"
+	damage = 55
+	armour_penetration = -35
```

Comment:
This indicates that this is from a very old downstream. We no longer use negative armor values. Instead, if you want to make something vulnerable to armor, you can instead do ``weak_against_armour = TRUE``. We haven't used negative values in I think like four years.

Reply:
Yeah, this has been in the works for a LONG while.